crates/meroctl/src/cli/context/event.rs

```diff
@@ -0,0 +1,109 @@
+use std::process::Command;
+
+use calimero_primitives::context::ContextId;
+use clap::Parser;
+use eyre::Result as EyreResult;
+use tokio::time::{sleep, Duration};
+
+use crate::cli::Environment;
+use crate::common::{do_request, fetch_multiaddr, load_config, multiaddr_to_url, RequestType};
+use crate::output::{ErrorLine, InfoLine};
+
+#[derive(Debug, Parser)]
+#[command(about = "Listen for context events and execute commands")]
+pub struct EventCommand {
+    #[arg(help = "The context ID to listen to")]
+    context_id: ContextId,
+
+    #[arg(help = "The shell command to execute when event occurs")]
+    command: String,
+
+    #[arg(
+        short,
+        long,
+        help = "Interval in seconds to check for events",
+        default_value = "1"
+    )]
+    interval: u64,
+
+    #[arg(
+        short,
+        long,
+        help = "Maximum number of events to trigger (0 for unlimited)",
+        default_value = "0"
+    )]
+    count: u32,
```

Comment:
```suggestion
        help = "Maximum number of events to trigger (0 for unlimited)",
        default_value_t
    )]
    count: u32,
```

I'd have preferred an `Option<NonZeroU32>` but I can think of some arguments in favor of what you did here, so it's fine for now